src/options/MapOptions.js

```diff
@@ -1192,7 +1242,7 @@ class MapOptions extends React.Component {
         }
 
         // Handle anomalies
-        if (tile['type'] === 'anomaly') {
+        if (tile['type'] === 'red') {
```

Comment:
Is this similar to the other PR you had? Where anomalies were anomalies were being used where reds were supposed to be?